src/components/menu/menu.svelte

```diff
@@ -0,0 +1,234 @@
+<script context="module" lang="ts">
+  import type { HTMLAttributes } from 'svelte/elements'
+  declare global {
+    namespace JSX {
+      interface IntrinsicElements {
```

Comment:
Should we move `leo-option` definiton here too?

Reply:
added